### ISSUE TITLE
Support for aliasing commands

### ISSUE THREAD

**Issue by alexdavid** [Originate/nifty#19]
From our talk today with @kevgo and @dhakiki

Allow aliasing commands

Possible API:
```coffeescript
module.exports = new Nifty.Command 'normalCommand', ->
  # ...


module.exports = new Nifty.Command ['potential', 'aliases', 'here'], ->
  # ...
```

What do you guys think?

**Comment by kevgo** [comment]
If your approach is that I can provide an array of names instead of a name while defining a command, then yes, that's the simplest possible approach here, and I like it.

The obvious alternative would be a dedicated aliasing API: 

```coffeescript
module.exports = new Nifty.Command 'getText', ->
  # ...
Nifty.alias 'getText', 'getTextOf'
```

The dedicated aliasing API would be more complex, but have the advantage that aliases can be defined later by the users. This would for example allow to define names for commands in other languages as part of I18N. In the spirit of YAGNI I'm fine postponing this and sticking with the simplest possible approach for now though.

**Comment by alexdavid** [comment]
Yeah, if a user really wants to alias a command they can do this:

```coffeescript
# Alias the getText command to getTextOf:
module.exports = new Nifty.Command 'getTextOf', ->
  @browser.getText.apply this, arguments
```

It's not as nice as `Nifty.alias 'getText', 'getTextOf'` but it's an option for now. We can add a nicer API if it becomes necessary.

**Comment by kevgo** [comment]
Ok, good enough for now.
